perun-openapi/openapi.yml

```
@@ -16416,6 +16416,35 @@ paths:
                 appId: { type: integer, description: "application id" }
                 reason: { type: string, description: "you can specify reason for case: mailType == APP_REJECTED_USER" }
 
+  /json/registrarManager/sendMessages:
+    post:
+      tags:
+        - RegistrarManager
+      operationId: sendMessages
+      summary: Manually re-sends mail notifications for existing applications.
+      description: |
+        Expected to be called as a result of direct VO administrator action in the web UI.
+      responses:
+        '200':
+          $ref: '#/components/responses/VoidResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              title: InputSendMessages
+              description: "input to sendMessages"
+              type: object
+              required:
+                - mailType
+                - ids
+              properties:
+                mailType: { $ref: '#/components/schemas/MailType' }
+                ids: { $ref: '#/components/parameters/ids' }
```

```suggestion
                ids: { type: array, items: { type: integer }, description: "ids of applications" }
```
something like this to make openapi parsing pass